iModelCore/ecobjects/PublicApi/ECObjects/ECSchema.h

```diff
@@ -3106,6 +3120,7 @@ struct ECSchema : RefCountedBase, IECCustomAttributeContainer
 friend struct ECSchemaDownConverter;
 
 friend struct SearchPathSchemaFileLocater;
+friend struct StringSchemaLocater;

```

Comment:
Why does StringSchemaLocater need to be a friend of ECSchema?

Reply:
I originally included StringSchemaLocater as a friend of ECSchema because SearchPathSchemaFileLocater was also a friend of ECSchema. However, this is not necessary, so I have removed StringSchemaLocater from the list of friends of ECSchema.